### PR TITLE
VOTE-1409 tracking tool updates

### PR DESCRIPTION
**Pull request by rayestrada**
## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-1409

## Description (optional)
This removes 2 modules from the codebase: Google Tag and crazyegg. After doing testing and research Google Tag module would be harder to maintain than the hardcoded value. Additionally, google tag can implement crazy egg remotely without the need for a module so we are removing crazyegg module from the codebase. Finally we are hardcoding in GTM code for logged out users.

### Post-deploy steps
1. run `lando retune`

### Testing steps
1. VIsit http://vote-gov.lndo.site/ open the inspector and look at the sources tab to confirm googletagmanager.com is there.
Screen Shot 2023-05-31 at 3 02 18 PM

### CODE REVIEW

**Review by SamiraMSadat — APPROVED** [review]
I have confirmed Google Tag Manager is running. This is good to merge after resolving the merge conflict.

**Comment by rayestrada** [comment]
Merge conflict is resolved

**Pull request merged**
